### PR TITLE
Add: Yunojuno's Django Developer Job

### PR DESCRIPTION
**Pull request by djm**
This is a re-attempt on behalf of Estella for #214. The position is still open.

Many thanks!

Darian

### CODE REVIEW

File: jobs/yunojuno-django-developer-london.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+* Excellent Python (2.x and ideally 3.x) and Django (1.8+)
+* Competent HTML, CSS and JavaScript (including jQuery
+
+# Bonus points for
+
+* ElasticSearch
+* Postgresql
+* Heroku
+* Docker
+* Vagrant
```
Comment (salimfadhley) [inline jobs/yunojuno-django-developer-london.html]: You can put these in the tags if you want.

**Comment by salimfadhley** [comment]
I've merged this - thanks for submitting. My regards to Estella.

File: jobs/yunojuno-django-developer-london.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+title: "Platform Developer - Python / Django"
+company: YunoJuno
+url: http://www.yunojuno.com
+location: London, England
+
+contract: permanent
+contact:
+    name: Estella Reed
+    email: estella@yunojuno.com
```
Comment (salimfadhley) [inline jobs/yunojuno-django-developer-london.html]: You can add a phone number if you want.

**Comment by djm** [comment]
Thanks @salimfadhley!

**Pull request merged**